recipes/gnuradio/meta.yaml

```
@@ -0,0 +1,275 @@
+{% set version = "3.8.0.0" %}
+
+package:
+  name: gnuradio
+  version: {{ version }}
+
+source:
+  url: https://github.com/gnuradio/gnuradio/releases/download/v{{ version }}/gnuradio-{{ version }}.tar.gz
+  sha256: 3a9c90111f22f2f6f30450731dc671ad28ce824fc1a7bc8ea0783da9b95a7092
+  patches:
+    - 0001-Use-raw-strings-for-paths-substituted-directly-into-.patch
+    - 0001-Don-t-use-hard-coded-path-constants.patch
+    - 0001-python-Do-not-link-against-python-lib-for-building-a.patch
+
+build:
+  number: 0
+  skip: True  # [win and py2k]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake >=3.8
+    - ninja
+    - pkg-config  # [not win]
+   # libudev needed to link against libusb on linux
+    - {{ cdt('libudev') }}  # [linux]
+   # below are needed to link with Qt for qtgui
+    - {{ cdt('libice') }}  # [linux]
+    - {{ cdt('libselinux') }}  # [linux]
+    - {{ cdt('libsm') }}  # [linux]
+    - {{ cdt('libx11') }}  # [linux]
+    - {{ cdt('libxau') }}  # [linux]
+    - {{ cdt('libxcb') }}  # [linux]
+    - {{ cdt('libxdamage') }}  # [linux]
+    - {{ cdt('libxext') }}  # [linux]
+    - {{ cdt('libxfixes') }}  # [linux]
+    - {{ cdt('libxrender') }}  # [linux]
+    - {{ cdt('libxxf86vm') }}  # [linux]
+    - {{ cdt('libuuid') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+
+  host:
+    - boost-cpp
+    - click
+    - click-plugins
+    - fftw
+    - gsl
+    # blas needed to link with gsl
+    - libblas
+    - libcblas
+    - libusb >=1
+    - log4cpp
+    - mako
+    - mpir
+    # we don't build against numpy C lib
+    # (only needed at runtime, except gnuradio CMake checks for it at build)
+    # specify version so pin is ignored
+    - numpy >=1.1
+    - python
+    - six
+    - swig
+    - volk >=2
+  # gnuradio.audio
+    - alsa-lib  # [linux]
+    - portaudio  # [linux]
+  # gnuradio.qtgui
+    - pyqt
+    - qt
+    - qwt
+  # gnuradio.uhd
+    - uhd
+  # gnuradio.zeromq
+    - cppzmq
+    - zeromq
+
+test:
+  requires:
+    - sqlite
+
+outputs:
+  - name: gnuradio-core
+    script: install_core.sh  # [not win]
+    script: install_core.bat  # [win]
+    build:
+      run_exports:
+        - {{ pin_subpackage('gnuradio-core', min_pin='x.x.x', max_pin='x.x.x') }}
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - cmake
+        - ninja
+        - pkg-config  # [not win]
+      host:
+        - alsa-lib  # [linux]
+        - boost-cpp
+        - fftw
+        - gsl
+        - libblas
+        - libcblas
+        - libusb
+        - log4cpp
+        - mpir
+        - portaudio  # [linux]
+        - python
+        - swig
+        - volk
+      run:
+        - boost-cpp
+        - click
+        - click-plugins
+        - fftw
+        - {{ pin_compatible('log4cpp') }}
+        - mako
+        - numpy
```

Is this only using the Python API? Or is the C API also used? If so this is needed here and elsewhere
```suggestion
        - {{ pin_compatible('numpy') }}
```

---

Only the Python API is used, and numpy is not actually a host dependency except that the build system checks that it is there to ensure the run dependency on its own, so the pin is not needed.